tox.ini

```diff
@@ -0,0 +1,17 @@
+[tox]
+envlist = py27,py35,pep8
```

Comment:
Travis matrix has py34 as well but that appears to be missing here. I think it would be good to test---can we add it here?